### ISSUE TITLE
Add and define workflow for the research and data page

### ISSUE THREAD

**Issue by fibasile** [fabfoundation/fabfoundation#8]
Research section- workflow for how people submit their papers and then put them up online.

**Comment by fibasile** [comment]
This should be investigated:

- one option is to create something like this:
  https://github.com/academany/awesome-fabacademy
  Where people basically contribute on the project forking it and sending 'pull requests'

- other option is to use something like reddit

**Comment by fibasile** [comment]
Found this tool:
https://i-librarian.net

It has a version that we can self-host, need to investigate further

**Comment by rebeccaottinger** [comment]
did not mean to close it!

**Comment by fibasile** [comment]
I prepared this:
https://forms.fabfoundation.org/publication/add

**Comment by rebeccaottinger** [comment]
Hi Fiore! I've added the link you made for submissions to the new website. I'm wondering what you think in terms of how to present the current and new submissions. Do you think we can go off of what is currently set up on the new website, or would need to create something totally different? Like the i-librarian or awesome-fabacademy... I will show her both of these on Wednesday. 
For now, is there a way to create the research page to be grouped by subject somehow, maybe with the accordion style tabs?

**Comment by fibasile** [comment]
As of now, there's no category associated to the materials, they are only grouped by being a pdf or not. We can use the database I setup for entering more information about each and starting to build an archive. There's some manual work, mainly writing descriptions and looking for publication data. 
Once this is in place we can have a much better interface in the website that pulls directly from the database.

**Comment by rebeccaottinger** [comment]
Hi Fiore! @fibasile 

Our summer intern, Angie Li, and I are going through to try out the form that you created for the research papers, submitting papers we already have info for from the master list Sherry sent. We came across a couple things...

1. What I can see from the submission is what you see below in this e-mail. 

When I clicked on the it 'view submission' form, it didn't have the other information we inputed: author, publication date, publication source, and an attachment with the pdf. 

2. Another thing to think about is the following: Right now Angie and I are filling in your research form with a few pdf's. if I put in my name and email, the submission looks like it's coming from me and not from the author (Neil, for example). I think that's ok, but  the Full Name and email should not be able to be viewed by the public when we put it on the website, because otherwise it will say Rebecca ottinger and my email, for example. Does that make sense? 

I don't think I should put Neil's Full Name and email in that spot, for example, since he didn't actually do the submitting of the paper. But of course I would put his name as the author. What do you think? 

I did talk with Sherry a little more about the look of the research section, and am going to confirm with her on Monday/Tuesday again and can send you our ideas. Angie and I are going to submit a few more papers today to have them in your repository and then we can go from there! 

3.  Can we make the summary box more than 300 characters max? 

Thank you!!

Rebecca

**Comment by fibasile** [comment]
All the data is saved correctly. The email contains just an extract of what is available in https://forms.fabfoundaiton.org/admin and what will be published on the site. (check the attached screenshot). 

The submitter info shouldn't be shown but is useful as internal reference! Who did send this, and how to reach out in case some information is missing.

I'll update the 300 char limit, but consider this needs to be shown in a search result, so don't make it too long.
![1f4f4d0e-d686-455f-b6a4-cbcc7ecccc12](https://user-images.githubusercontent.com/32947/58894490-5f43d780-86f2-11e9-8cfa-e3e940e8fb76.jpeg)

**Comment by fibasile** [comment]
So this is how the new search / index block could look like. This is working with the real data. http://fibasile.fabcloud.io/test-research-page/#

@rebeccaottinger

**Comment by fibasile** [comment]
Forgot to mention that I added a checkbox, "visible" for the publications in the admin.

So when you receive a notification of a new paper, you go in the admin and set this checkbox to true to publish it on the website, or it will not be shown because we don't want content to appear on the FF website without moderation.

**Comment by rebeccaottinger** [comment]
Looks great! Gong to start submitting more today...

**Comment by rebeccaottinger** [comment]
@fibasile I'm starting to upload! Trying to upload websites and it keeps saying server error, specifically for when I choose website as the type of publication... 

any thoughts?

Also, I added a category called technology... I think we should have a category for economy/industry as well, but I don't want to add too many!

**Comment by fibasile** [comment]
@rebeccaottinger the error is now fixed, it was related to empty published date, now it is handled correctly.

Two things:
- you can use the admin to add the paper (is much more convenient to use for bulk data entry) 

- you need to publish them once submitted, use this checkbox and save:
![image](https://user-images.githubusercontent.com/32947/59144329-aeca2200-89d6-11e9-994b-658d3da2f7af.png)

**Comment by fibasile** [comment]
You can keep working on it.
I'm leaving the debug on with verbose errors, so if you get any error you can take a screenshot and paste it here.

**Comment by rebeccaottinger** [comment]
<img width="1110" alt="Screen Shot 2019-06-14 at 2 03 22 PM" src="https://user-images.githubusercontent.com/48486398/59528812-433bf500-8ead-11e9-99f5-d97d9f3e078d.png">

I just tried to put in a website again. I only put in information were there is an * in the field (title, type, category, region), as I assume the * means it is required to submit, and the other options are not necessary... that's what people coming to the site will most likely think as well

Are there more fields that have to be filled in in order for it to go through no matter the type of publication, website or pdf, etc? If there is, we just have to make sure that's clear.

Thanks! @fibasile

**Comment by fibasile** [comment]
No, it was an error from my side (still linked to the publication date). Should be fixed now. All the mandatory fields are marked with a *.
